### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KasraAlizadeh**
The test server Rest API with the **GET** method was developed.
I thank you for reviewing this pull request.

### CODE REVIEW

File: serversideapi/src/main/java/com/example/serversideapi/ServersideapiApplication.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+// Main class of api for execution
```
Comment (seagull-dev-team) [inline serversideapi/src/main/java/com/example/serversideapi/ServersideapiApplication.java:6]: Really? Who doesn't know that `@SpringBootApplication` is "Main class of api for execution"

File: serversideapi/src/main/java/com/example/serversideapi/GreetingController.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 import java.util.concurrent.atomic.AtomicLong;
 
+// create structure of controller
```
Comment (seagull-dev-team) [inline serversideapi/src/main/java/com/example/serversideapi/GreetingController.java:9]: isn't `@RestController` enough?

File: serversideapi/src/main/java/com/example/serversideapi/Greeting.java [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package com.example.serversideapi;
 
+// Create structure of api
```
Comment (seagull-dev-team) [inline serversideapi/src/main/java/com/example/serversideapi/Greeting.java:3]: Kidding Me? Of course it's a POJO and doesn't need additional comment

File: serversideapi/src/main/java/com/example/serversideapi/ServersideapiApplication.java (hunk 1 above)
Comment (KasraAlizadeh) [reply serversideapi/src/main/java/com/example/serversideapi/ServersideapiApplication.java:6]: Thanks for the feedback.
I'll change my comments structure

File: serversideapi/src/main/java/com/example/serversideapi/Greeting.java (hunk 3 above)
Comment (KasraAlizadeh) [reply serversideapi/src/main/java/com/example/serversideapi/Greeting.java:3]: Okay:)
I'll change it

File: serversideapi/src/main/java/com/example/serversideapi/GreetingController.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+@RestController
+public class GreetingController {
+
+    private static final String template = "Hello, %s!";
+    private final AtomicLong counter = new AtomicLong();
```
Comment (seagull-dev-team) [inline serversideapi/src/main/java/com/example/serversideapi/GreetingController.java:14]: OMG? Why on earth you need an `AtomicLong` on a project like this

**Pull request merged**